### ISSUE TITLE
construct() does not pop scope stack.

### ISSUE THREAD

**Issue by sisshiki1969** [maekawatoshiki/rapidus#24]
test code

```javascript
var a = 0
new f()
console.log(a)

function f() {
  var a = 10
}
```
```
$ node test.js
0
$ cargo run test.js
    Finished dev [unoptimized + debuginfo] target(s) in 0.25s
     Running `target/debug/rapidus examples/test.js`
10
```

**Comment by maekawatoshiki** [comment]
thank you for your report

**Comment by maekawatoshiki** [comment]
This bug is fixed (#25)
